data/tokens.json

```diff
@@ -1499,30 +1499,6 @@
       }
     }
   },
-  "Accordion": {
-    "gtmds_helper_accordion_border_open": {
-      "value": {
-        "color": "{gtmds_color_gray-4}",
-        "width": "{gtmds_border-width-base}"
-      },
-      "type": "border"
-    },
-    "gtmds_helper_accordion_heading_border_open": {
-      "value": {
-        "color": "{gtmds_color_gray-4}",
-        "width": "{gtmds_border-width-base}"
-      },
-      "type": "border"
-    },
-    "gtmds_helper_accordion_color_background_hover": {
-      "value": "{gtmds_color_blue-1}",
-      "type": "color"
-    },
-    "gtmds_helper_accordion_heading_border_radius": {
-      "value": "{gtmds_border-radius-md}",
-      "type": "borderRadius"
-    }
-  },
```

Comment:
@tsnikolov it is a good practice to also define the typography's tokens. Since we don't know the content yet, the title is enough I believe.